docs/io.cozy.files.md

```diff
@@ -27,14 +27,29 @@ The attributes of a file are:
 - `name`: {string} the file name
 - `trashed`: {bool} true if the file is in the trash
 - `md5sum`: {string} the checksum of its content, computed with the MD5 algorithm
-- `created_at`: {timestamp} the date of the creation of this directory
-- `updated_at`: {timestamp} the date of the last update of this directory
+- `created_at`: {date} the date of the creation of this directory
+- `updated_at`: {date} the date of the last update of this directory
 - `tags`: {array of strings} a list of tags
 - `size`: {number} the size of its content, in bytes
 - `executable`: {bool} true is the file has the executable bit on UNIX (`chmod +x`)
 - `class`: {string} a class in the list: `['image', 'document', 'audio', 'video', 'text', 'binary']`
 - `mime`: {string} the full mime-type
 - `metadata`: {map} an optional map of metadata (for example `width`, `height`, and `datetime` for an image)
+  - `width`: {number}
+  - `height`: {number}
+  - `datetime`: {date} : date in original image file metadata
+  - `gps`: {map} : localization metadata with the following attributes
```

Comment:
As GPS is a particular localization system, I think it would be preferable to have something generic like `location` or `loc`.

Reply:
Too late, it is already done by the stack: this information is extracted from the EXIF metadata, and the EXIF fields for that was GPSLatitude and GPSLongitude, so I've used `gps` as the key to copy what is proposed by EXIF.

Reply:
If we had to choose the wording today, I will agree with you. But it was done 2 years ago: https://github.com/cozy/cozy-stack/commit/a310a19f6f088e9d860b3114af568b745b348ca2#diff-31424a8a18e905d88115229a6ca31d10R102

Reply:
Oh, sorry, I thought this was an addition in the io.cozy.files document.

Let's leave that as is then